boto/dynamodb2/table.py

```diff
@@ -439,7 +446,134 @@ def update(self, throughput, global_indexes=None):
         self.connection.update_table(
             self.table_name,
             provisioned_throughput=data,
-            global_secondary_index_updates=gsi_data
+            global_secondary_index_updates=gsi_data,
+        )
+        return True
+
+    def create_global_secondary_index(self, global_index):
+        """
+        Creates a global index in DynamoDB after the table has been created.
+
+        Requires a ``global_indexes`` parameter, which should be a
+        ``GlobalBaseIndexField`` subclass representing the desired index.
+
+        Returns ``True`` on success.
+
+        Example::
+
+            # To create a global index
+            >>> users.create_global_secondary_index(
+            ...     global_index=GlobalAllIndex(
+            ...         'TheIndexNameHere', parts=[
+            ...             HashKey('requiredHashkey', data_type=STRING),
+            ...             RangeKey('optionalRangeKey', data_type=STRING)
+            ...         ],
+            ...         throughput={
+            ...             'read': 2,
+            ...             'write': 1,
+            ...         })
+            ... )
+            True
+
+        """
+
+        gsi_data = None
+        gsi_data_attr_def = None
+
+        if global_index:
```

Comment:
The global index should not be null – otherwise you shouldn't update the table. Not sure what @danielgtaylor would prefer for this, but it seems like `if not global_index`, just return False and log the fact that it's empty?